packages/payments-plugin/src/mollie/mollie.plugin.ts

```diff
@@ -21,6 +21,14 @@ export interface MolliePluginOptions {
      * This is used by Mollie to send webhook events to the Vendure server
      */
     vendureHost: string;
+
+    /**
+     * @description
+     * For backwards compatibility, by default set to false.
+     * When enabled, the `redirectUrl` can be passed via the `createPaymentIntent` mutation 
+     * instead of being configured in the Payment Method.
+     */
```

Comment:
You can add 

```
@default false
@since 2.0.0
```

Reply:
It's being merged to minor right? But still only @since 2.0.0 ? Would expect that to be the next minor release actually?